docs/api/StreamClose.md

```diff
@@ -17,11 +17,17 @@ void
 
 # Parameters
 
-**TODO**
+`Stream`
+
+A Stream handle from a previous call to [StreamClose](StreamClose.md).
 
 # Remarks
 
-**TODO**
+The application should only close a stream after it has been completely shut down or it was never successfully started. Closing a stream before it has been completely shut down produces **undefined behavior** because clean up of the stream **must** be reflected on the wire with an application specific error code. When the app closes a stream without first shutting down, MsQuic has to "guess" at the error code to use (currently uses `0`) when sending the state change to the peer.
```

Comment:
```suggestion
The application should only close a stream after it has been completely shut down or it was never successfully started. Closing a stream before it has been completely shut down produces **undefined behavior** because clean up of the stream **must** be reflected on the wire with an application specific error code. When the app closes a stream without first shutting down, MsQuic has to guess which error code to use (currently uses `0`) when sending the state change to the peer.
```